source/postprocess/fluid_velocity_statistics.cc

```diff
@@ -0,0 +1,173 @@
+
+                const std::shared_ptr<MaterialModel::MeltOutputs<dim>> fluid_out
+                  = out.template get_additional_output_object<MaterialModel::MeltOutputs<dim>>();
+
+                for (unsigned int q = 0; q < n_q_points; ++q)
+                  {
+                    const Tensor<1,dim> gravity = this->get_gravity_model().gravity_vector(fe_values.quadrature_point(q));
+                    const double porosity = std::max(in.composition[q][porosity_idx], 1e-10);
+                    const double solid_density = out.densities[q];
+                    const double fluid_density = fluid_out->fluid_densities[q];
+                    const double fluid_viscosity = fluid_out->fluid_viscosities[q];
+                    const double permeability = fluid_out->permeabilities[q];
+                    const Tensor<1,dim> darcy_velocity = solid_velocity_values[q] -
+                                                         (permeability / fluid_viscosity / porosity) *
+                                                         gravity * (solid_density - fluid_density);
+                    local_max_fluid_velocity = std::max (darcy_velocity.norm(),
+                                                         local_max_fluid_velocity);
+                  }
+              }
+          }
+
+      const double global_max_fluid_velocity
+        = Utilities::MPI::max (local_max_fluid_velocity, this->get_mpi_communicator());
+
+      const std::string units = (this->convert_output_to_years() == true) ? "m/year" : "m/s";
+      const double unit_scale_factor = (this->convert_output_to_years() == true) ? year_in_seconds : 1.0;
+      const std::string advection_method = output_melt_velocity ? "melt" : "Darcy";
+      const std::vector<std::string> column_name = {"Max. " + advection_method + " velocity (" + units + ")"
+                                                   };
+
+      statistics.add_value (column_name[0],
+                            global_max_fluid_velocity * unit_scale_factor);
+
+      // also make sure that the other columns filled by this object
+      // all show up with sufficient accuracy and in scientific notation
+      for (auto &column : column_name)
+        {
+          statistics.set_precision (column, 8);
+          statistics.set_scientific (column, true);
+        }
+
+      std::ostringstream output;
+      output.precision(3);
+      output << global_max_fluid_velocity *unit_scale_factor
+             << ' ' << units;
+
+      return std::pair<std::string, std::string> ("Max " + advection_method + " velocity:",
+                                                  output.str());
+    }
+  }
+}
+
+
+// explicit instantiations
+namespace aspect
+{
+  namespace Postprocess
+  {
+    ASPECT_REGISTER_POSTPROCESSOR(FluidVelocityStatistics,
+                                  "fluid velocity statistics",
+                                  "A postprocessor that computes the root mean square and "
```

Comment:
your changelog entry and this description is not consistent with your implementation. The implementation only computes the max velocity, but the other two places say RMS and max velocity. Take a look over the whole PR and unify.